server/app/routes/patient.routes.js

```diff
@@ -99,7 +99,7 @@ router.get(
   Patient.getDocuments
 );
 router.put(
-  "/patient/:id/documents/:id",
+  "/patient/:id/documents/:id/status/:type",
```

Comment:
Technically we should add a validation here because we are not interested on any type. For example if I pass `/patient/${patient_id}/documents/${document_id}/status/ruhul` then status become `ruhul`. Also route path is not correct as well. 
We should always keep `/patient/:id/documents/:id` as route path but need to pass status as data.

Reply:
Acknowledged.